e2e/test/scenarios/onboarding/search/search.cy.spec.js

```diff
@@ -263,6 +290,57 @@ describe("scenarios > search", () => {
           });
         });
       });
+
```

Comment:
Possibly unrelated to this PR, but the bottom padding is evidently missing here (or the top padding is redundant):

![image](https://github.com/metabase/metabase/assets/6830683/187184ec-996a-45e9-beee-0bb4ee7fe7bd)

It's more prominent when hovering the only search result:

![image](https://github.com/metabase/metabase/assets/6830683/02986e2b-16f3-467a-b17c-5a2697f91dcd)

Reply:
Yeah this is going to be improved when we update the search result elements in general - just focusing on the sidebar for now